PixelWizards/StreamingImageSequence
# fix: fix bugs in handling memory overflow (#84)

* Eliminate map duplication in ImageCatalog::GetImageMap()
* When unloading image data, don't change anything if the RawData is null.
* Fix bug when setting the start pos of the current "order"  (ImageCollection::m_curOrderStartPos)
* Reset internal values (m_latestRequestFrame) when UnloadAllImages() API is called.
* Add new status READ_STATUS_UNAVAILABLE, and rename READ_STATUS_NONE to READ_STATUS_IDLE
* Add constructor in ImageCollection
* Update cs code to correspond to the plugin code changes
* Update DLL
* Update Mac bundle

Tests:
* Add a test for loading multiple images
* Add a test for testing automatic memory deallocation for unused images
* Add a test if we stop loading images if the memory is not enough for the current frame.
* Add a test that combines all the tests above in the same test case.
* Set the maximum of allowed memory for images during testing
* Gather memory cleanup check in one function. TestUtility::CheckMemoryCleanup()
* Move test utility functions to TestUtility class

Others:
* Update test project version to 2019.4.0f1
* Update scenes in the test project
* Minor namespace fixes
